editor/src/messages/portfolio/document/node_graph/document_node_definitions.rs

```diff
@@ -2005,6 +2005,103 @@ fn static_nodes() -> Vec<DocumentNodeDefinition> {
 			description: Cow::Borrowed("Convert vector geometry into a polyline composed of evenly spaced points."),
 			properties: Some("sample_polyline_properties"),
 		},
+		DocumentNodeDefinition {
```

Comment:
You don't usually need to manually define the document node definition as it can be automagically generated.